src/prefect/flows.py

```diff
@@ -62,7 +62,8 @@
 from prefect.utilities.hashing import file_hash
 from prefect.utilities.importtools import import_object
 
-T = TypeVar("T")  # Generic type var for capturing the inner return type of async funcs
+# Generic type var for capturing the inner return type of async funcs
+T = TypeVar("T")
```

Comment:
Ideally we'd retain this to match the other type variables here

Reply:
I am sorry, I don't quite get this.

Reply:
Ah. You changed the position of this comment so it doesn't match the comments below it for the other type variables. This change isn't necessary for this pull request and you should undo it.

Reply:
Hmm, I have no idea how it got changed, I think somehow black did it, anyways I am reverting it back.